src/crypto/convert.js

```diff
@@ -36,7 +36,7 @@ export function bufferToHex(buffer) {
  */
 
 export function hexToBuffer(hex) {
-	return naclInstance.from_hex(hex);
+	return Buffer.from(naclInstance.from_hex(hex));
```

Comment:
`Buffer.from(hex, 'hex')` - no need to use the `naclInstance` if we're only going to convert the result anyway.